### ISSUE TITLE
It would be nice to be able to install Studio via flatpak from flathub

### ISSUE THREAD

**Issue by phozzy** [libfive/libfive#250]
### Description

[Description of the issue]
It would be nice to be able to install Studio via flatpak from https://flathub.org/home
Is it possible to make "oficial" flatpak?
[Steps to reproduce]
Open site https://flathub.org/home try to search for Studio.
[What did you expect to happen?  What happened?  All the time?]
I expect to be able to install Studio with:
`flatpak install flathub com.libfive.Studio`
[Are there any error messages printed to the console?]

### Versions

- Operating system: Any linux with flatpak installed
- `libfive` commit hash: not related
  - Is this the latest commit to `master`?  not related
- Qt version (if applicable): not related

### Additional Information

Any additional information, configuration or data that might be necessary to reproduce the issue.

**Comment by mkeeter** [comment]
Yup, I agree that this would be nice!  I'm not going to take up the role of flatpak maintainer, but I'd be happy to assist if someone wants to step up.
